tools/esdlcmd/esdlcmd_core.cpp

```diff
@@ -328,17 +328,20 @@ class Esdl2XSDCmd : public EsdlHelperConvertCmd
 
         generateNamespace(tns);
 
-        params->setProp( "tnsParam", tns );
-        params->setProp( "optional", optOptional );
+        // All params are treated as expressions, so any strings must be quoted
+        // 1/0 are not equivalent to true/false, as 0 evaluates to true
+
+        params->setProp( "tnsParam", StringBuffer('\'').append(tns).append('\'') );
```

Comment:
I would use VStringBuffer("'%s'", tns.str()) instead of StringBuffer('\'').append(tns).append('\'').

Reply:
Done, wrapped in macro.